src/getting_started.md

```diff
@@ -4,18 +4,31 @@ This guide walks you through the steps needed to get the Alan compiler and runti
 
 ## Installation
 
-Node.js is required, with a minimum version of 10.20.1, to install `alan-compile`, run:
+For users on Windows, MacOS, and Ubuntu Linux with x86-64 processors, you can [download the Alan executable](https://github.com/alantech/alan/releases), extract it into your `$PATH`/`%PATH%` (eg `/usr/local/bin` on MacOS and Ubuntu, and `C:\Windows` on Windows if you don't want to add a new executable path to your environment variable), and you're ready to go!
 
-```bash
-npm i -g alan-compile
-```
+### Source Installation
+
+If your operating system and/or CPU architecture do not match the above, you'll need a development environment to build Alan locally.
 
-Rust is required, with a minimum version of 1.41.1 to install `alan-runtime`, run:
+You'll need:
+
+* git (any recent version should work)
+* Node.js >=10.20.1, <14.0.0
+* Rust >=1.41.1
+* A complete C toolchain (gcc, clang, msvc)
+* Python >=2.7, <3.0 (and named `python2` in your PATH)
+
+Once those are installed, simply:
 
 ```bash
-cargo install alan-runtime
+git clone https://github.com/alantech/alan
+cd alan
+make
+sudo make install
 ```
 
+This may take a few hours.
```

Comment:
What why?? Once dependencies are installed it should be fine?

Reply:
If you're not on an x86-64 platform, or you're not on Windows, Mac, or Ubuntu, it is unlikely that your computer will find an `nexe` prebuilt image from here: https://github.com/nexe/nexe/releases/tag/v3.3.3

And so it will have to build from scratch, and that will take hours.

Reply:
So this either needs to be expanded on or removed and I vote for the latter since it seems like an edge case. Also at the beginning of this section someone could to a source installation to contribute to Alan.